ComSemApp/student/views.py

```diff
@@ -669,24 +669,39 @@ def form_valid(self, form):
 # ALL THESE CLASSES SHOULD IMPLEMENT DIFFERENT VIEWS LATER
 # the foo() functions are just placeholders too keep the classes in here
 class SpeakingPracticeView(StudentViewMixin, CourseViewMixin, TemplateView):
+    """
+      Serves the content of the speaking practice page which presents problems
+      and recieves user input.
+    """
     template_name: str = 'ComSemApp/student/assessment.html'
 
     def foo():
         pass
 
 class SpeakingPracticeGeneratorView(StudentViewMixin, CourseViewMixin, TemplateView):
+    """
+      Serves the content of the speaking practice page which lists expressions and
+      prompts user to select the onse they want to practice.
```

Comment:
Ones is misspelled here.

Reply:
Fixed in commit 510cfee9f4a625d56c85dae267771f5564f3775b